Source Code/Ticket Management System/user.c

```diff
@@ -109,17 +109,17 @@ void user_setEmail(struct User* user, char* email)
 		}
 		else if (strlen(email) != strlen(user->email))
 		{
-			wchar_t* old_email = user->email;
-			user->email = realloc(user->email, strlen(email) + 1);
+			char* old_email = user->email;
+			user->email = (char*)realloc(user->email, strlen(email) + 1);
 			if (user->email == NULL)
 			{
 				user->email = old_email;
 				return;
 			}
-			else if (old_email != user->email)
+			/*else if (old_email != user->email)
```

Comment:
Why?

Reply:
I talked with you about it, it doesn't work so I just put it as a comment for the time being..

Reply:
Yes, but why doesn't it work?